### ISSUE TITLE
Does not render minDate and maxDate change

### ISSUE THREAD

**Issue by serkanekinci** [stephy/CalendarPicker#272]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-calendar-picker@7.0.9` for the project I'm working on because [Insert reason here].

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-calendar-picker/CalendarPicker/index.js b/node_modules/react-native-calendar-picker/CalendarPicker/index.js
index c225330..c1dfb73 100644
--- a/node_modules/react-native-calendar-picker/CalendarPicker/index.js
+++ b/node_modules/react-native-calendar-picker/CalendarPicker/index.js
@@ -74,9 +74,7 @@ export default class CalendarPicker extends Component {
 
     let minMaxDates = {};
-    if (prevProps.minDate !== this.props.minDate ||
-        prevProps.minDate !== this.props.minDate
-    ) {
+    if (prevProps.minDate !== this.props.minDate || prevProps.maxDate !== this.props.maxDate) {
       minMaxDates.minDate = this.props.minDate && moment(this.props.minDate);
       minMaxDates.maxDate = this.props.maxDate && moment(this.props.maxDate);
+
+      doStateUpdate = true
     }
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by peacechen** [comment]
Thanks @serkanekinci for finding this bug. Please submit a PR.

**Comment by peacechen** [comment]
Published in 7.1.1.

To speed up fixes in the future, please submit a PR instead of a patch file.
